src/linear_algebra/general_linear_group.lean

```diff
@@ -0,0 +1,199 @@
+/-
+Copyright (c) 2020 Anne Baanen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+
+The General Linear group $GL(n, R)$
+-/
```

Comment:
```suggestion
/-
Copyright (c) 2020 Chris Birkbeck. All rights reserved.
Released under Apache 2.0 license as described in the file LICENSE.
Authors: Chris Birkbeck
-/
```

Reply:
Oops, thank you. I guess you can see I just copied this over from another file in mathlib.

Reply:
And its 2021 now :D